src/controller/ReadInteraction.h

```diff
@@ -64,7 +64,16 @@ CHIP_ERROR ReportAttribute(Messaging::ExchangeManager * exchangeMgr, EndpointId
     auto readClient =
         chip::Platform::MakeUnique<app::ReadClient>(engine, exchangeMgr, callback->GetBufferedCallback(), readParams.mReportType);
 
```

Comment:
Pre-existing, but while we are here: need to null-check readClient.

Reply:
Done, thanks